docs/src/introduction/overview.md

```diff
@@ -1 +1,11 @@
-[WIP]
+# Overview
+> [WIP]
+
+Polygon Miden is a bi-directional token bridge and state machine. Miden Nodes act as operators that keep the state and compress state transitions recursively into STARK-proofs. The token bridge on Ethereum verifies these proofs. Users can run Miden clients to send RPC requests to the Miden Nodes to update the state.
```

Comment:
"Polygon Miden is a bi-directional token bridge and state machine." I wonder if it's better to communicate Miden as a "an Etherem based layer 2 zk-rollup."

Reply:
Yes, I think that's better - but I think we should also mentioned how it plugs into common Polygon ecosystem. Maybe we should use phrasing similar to the one we use in blog posts?

Reply:
Good point will do